_pytest/config.py

```diff
@@ -399,13 +399,15 @@ def consider_conftest(self, conftestmodule):
             self.consider_module(conftestmodule)
 
     def consider_env(self):
-        self._import_plugin_specs(os.environ.get("PYTEST_PLUGINS"))
+        specs = os.environ.get("PYTEST_PLUGINS")
+        if specs:
+            plugins = specs.split(',')
```

Comment:
It feels to me `import_plugin` is the better place for it because all other functions which import a plugin eventually call it, so don't miss marking plugins for assertion rewrite:

```
consider_pluginarg -> import_plugin
consider_conftest -> consider_module -> _import_plugin_specs -> import_plugin
consider_module -> _import_plugin_specs -> import_plugin
```

So I think this actually also fixes `consider_pluginarg`.

Could you provide an example were this would be a behavior change?